parachains/runtimes/bridge-hubs/bridge-hub-kusama/src/lib.rs

```diff
@@ -589,14 +599,85 @@ impl_runtime_apis! {
 		fn dispatch_benchmark(
 			config: frame_benchmarking::BenchmarkConfig
 		) -> Result<Vec<frame_benchmarking::BenchmarkBatch>, sp_runtime::RuntimeString> {
-			use frame_benchmarking::{Benchmarking, BenchmarkBatch, TrackedStorageKey};
+			use frame_benchmarking::{Benchmarking, BenchmarkBatch, BenchmarkError, TrackedStorageKey};
 
 			use frame_system_benchmarking::Pallet as SystemBench;
 			impl frame_system_benchmarking::Config for Runtime {}
 
 			use cumulus_pallet_session_benchmarking::Pallet as SessionBench;
 			impl cumulus_pallet_session_benchmarking::Config for Runtime {}
 
+			use xcm::latest::prelude::*;
+			use xcm_config::KsmRelayLocation;
+
+			impl pallet_xcm_benchmarks::Config for Runtime {
+				type XcmConfig = xcm_config::XcmConfig;
+				type AccountIdConverter = xcm_config::LocationToAccountId;
+				fn valid_destination() -> Result<MultiLocation, BenchmarkError> {
+					Ok(KsmRelayLocation::get())
+				}
+				fn worst_case_holding() -> MultiAssets {
+					// just concrete assets according to relay chain.
+					let assets: Vec<MultiAsset> = vec![
+						MultiAsset {
+							id: Concrete(KsmRelayLocation::get()),
+							fun: Fungible(1_000_000 * UNITS),
+						}
+					];
```

Comment:
This does not look anywhere near like a worst case scenario. Does the bridge hub support any other kinds of asset other than the relay chain native token? If not, then this makes sense, otherwise the point is to fill up the holding register with assets, up to the maximum specified in the XCM configs.

Reply:
BridgeHub will support only relay chain native token (no pallet_assets here), thats why I removed here `HOLDING_FUNGIBLES/HOLDING_NON_FUNGIBLES`